dev/com.ibm.ws.cdi.extension.bce_fat/fat/src/com/ibm/ws/cdi/extension/bce_fat/basic/app/permissions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<permissions xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
+        http://xmlns.jcp.org/xml/ns/javaee/permissions_7.xsd"
+    version="7">
```

Comment:
Similarly, I know this does not _need_ to be EE10 but would be useful to test
```suggestion
    xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee
        https://jakarta.ee/xml/ns/jakartaee/permissions_10.xsd"
    version="10">
```

Reply:
Ah, I didn't realise there was an EE10 version of the permissions schema. The EE9.1 platform spec still links to the 7 version.

Reply:
https://jakarta.ee/xml/ns/jakartaee/ shows you what they look like for each of the releases.

Reply:
It looks like we don't yet support version 10 of permissions.xml so I've set it to 9.

Reply:
BTW, support is on its way.  Not sure if it will be here next week or after the GM build on March 2.